index.html

```diff
@@ -0,0 +1,275 @@
+							 	</p>
+							 	<div class="user-rating">
+							 		<i class="rating_icon fa fa-thumbs-up"> 24 </i>
+							 		<i class="rating_icon fa fa-thumbs-down"> 2</i>
+							 		<span>REPLY</span>
+							 	</div>
+							</div>
+				</div>
+		</section>
+		<aside class="recommendation right">
+				<div class="upnext">
+					<a href="http://"><span class="left">Up next</span></a>
+					 <a href="http://"><span class="right">AUTOPLAY</span></a>
+				</div>
+				<div class="video border-bottom">
+					<a href="http://">
+						<img src="preview_new.png" class="left" alt="Video preview">
+						<p class="left">
+							<strong class="">
+							Contributing to Open Source
+							<br>Part II: The Real Way
+							</strong>
+						</p>
+						<p class="left rec_caption">
+							The Odin Project <br>
+							10 views
+						</p>
+					</a>
+				</div>
+
+				<div class="video">
+					<a href="http://">
+						<img src="preview_new.png" class="left" alt="Video preview">
+						<p class="left">
+							<strong class="">
+							Contributing to Open Source
+							<br>Part II: The Real Way
+							</strong>
+						</p>
+						<p class="left rec_caption">
+							The Odin Project <br>
+							10 views
+						</p>
+					</a>
+				</div>
+				<div class="video">
+					<a href="http://">
+						<img src="preview_new.png" class="left" alt="Video preview">
+						<p class="left">
+							<strong class="">
+							Contributing to Open Source
+							<br>Part II: The Real Way
+							</strong>
+						</p>
+						<p class="left rec_caption">
+							The Odin Project <br>
+							10 views
+						</p>
+					</a>
+				</div>
+				<div class="video">
+					<a href="http://">
+						<img src="preview_new.png" class="left" alt="Video preview">
+						<p class="left">
+							<strong class="">
+							Contributing to Open Source
+							<br>Part II: The Real Way
+							</strong>
+						</p>
+						<p class="left rec_caption">
+							The Odin Project <br>
+							10 views
+						</p>
+					</a>
+				</div>
+				<div class="video">
+					<a href="http://">
+						<img src="preview_new.png" class="left" alt="Video preview">
+						<p class="left">
+							<strong class="">
+							Contributing to Open Source
+							<br>Part II: The Real Way
+							</strong>
+						</p>
+						<p class="left rec_caption">
+							The Odin Project <br>
+							10 views
+						</p>
+					</a>
+				</div>
+				<div class="video">
+					<a href="http://">
+						<img src="preview_new.png" class="left" alt="Video preview">
+						<p class="left">
+							<strong class="">
+							Contributing to Open Source
+							<br>Part II: The Real Way
+							</strong>
+						</p>
+						<p class="left rec_caption">
+							The Odin Project <br>
+							10 views
+						</p>
+					</a>
+				</div>
+
+				<button class="btn show-more">show more</button>
+		</aside>			
```

Comment:
please don't forget to clear the floats after using them.

Reply:
All correction effected thanks.